Docs/Verbali Interni/VI_2024-12-13/content.tex

```diff
@@ -0,0 +1,50 @@
+
+\section{Resoconto}
+% insert here all the steps
+\subsection{Metriche}
+Ci siamo chiesti se la sezione delle metriche dovesse essere riportata come sezione autonoma o come sottosezione all'interno delle \textit{Norme di progettto}, abbiamo deciso che inseriremo una sezione dedicata alle metriche.
+
+\subsection{Sprint 4}
+Dopo aver evidenziato che lo sprint successivo si svolgerà nel periodo che va dal 23 dicembre 2024 al 3 gennaio 2025, e quindi in concomitanza con le festività natalizie, ci siamo confrontati per discutere le disponibilità di ciascun membro del team. Ogni partecipante ha espresso la propria disponibilità a lavorare al progetto durante lo sprint, e abbiamo deciso che i ruoli assegnati e le rispettive ore di lavoro saranno definiti in base alle disponibilità individuali, al fine di avviare il lavoro sul PoC il prima possibile.\\
+Abbiamo discusso anche i criteri per l'assegnazione dei ruoli nel prossimo sprint. In particolare, il ruolo di programmatore sarà affidato ai membri che hanno acquisito maggiore dimestichezza con le tecnologie richieste dal progetto, ovvero React e Three.js, e che daranno la propria disponibilità a lavorare nel periodo dal 23 dicembre 2024 al 3 gennaio 2025.  
+
+\subsection{Diagrammi UML dei Casi d'uso}
+Sono emerse alcune incertezze da parte dei membri del team sulla qualità dei diagrammi UML relativi ai casi d'uso e sulla loro adeguatezza per una corretta analisi dei requisiti. Per affrontare questi dubbi, si è deciso di fissare due incontri: uno con il proponente e uno con il Prof. Cardin. L'obiettivo è ottenere un riscontro sul lavoro svolto, porre eventuali domande per migliorare la qualità del nostro lavoro e proseguire al meglio con la stesura dei casi d'uso.  
+
+\subsection{Automazione Glossario}
+Durante la discussione sul glossario, è emersa la necessità di prevedere alcuni giorni prima della consegna dell'RTB per testare il funzionamento dell'inserimento automatico dei pedici. 
+\\Considerando la possibilità che l'automazione possa fallire e non si riuscisse a risolvere il problema in tempo, il team ha stabilito che, come soluzione alternativa, si procederà manualmente con l'inserimento dei pedici per garantire comunque la consegna puntuale dei documenti. 
+Contestualmente, è stato deciso di assegnare alta priorità alla risoluzione del problema dell'automazione subito dopo la consegna, per evitare simili inconvenienti in futuro.
+
+\subsection{Organizzazione directory del repository}
+Per garantire maggior organizzazione e struttura all'interno del repository, è stato deciso che, dopo la consegna dell'RTB, il repository verrà riorganizzato per garantire una chiara distinzione tra il lavoro svolto per l'RTB e quello per la PB. \\In particolare, la documentazione e i prodotti software relativi alle due baseline saranno separati in cartelle dedicate, così da migliorare l'ordine e la tracciabilità del lavoro.
```

Comment:
--Direi di aggiungere che anche la Github Pages sarà riorganizzato in questo modo

-- Non sarebbe piuttosto "dopo la consegna DELLA RTB" ?